package.json

```diff
@@ -6,20 +6,18 @@
     "node": "18.x"
   },
   "scripts": {
-    "build": "run-s build:clean && run-p build:js:prod build:common",
-    "build:clean": "shx rm -rf network-api/networkapi/frontend && shx mkdir -p network-api/networkapi/frontend/_js",
-    "build:common": "run-p build:images build:sass build:tailwind build:contribute",
+    "build": "run-s build:clean && run-p build:common build:js:prod && npm run postcss:build",
+    "build:clean": "shx rm -rf network-api/networkapi/temp/frontend && shx rm -rf network-api/networkapi/frontend && shx mkdir -p network-api/networkapi/frontend/_js",
+    "build:common": "run-p build:images build:css build:contribute",
     "build:contribute": "shx cp contribute.json network-api/networkapi/frontend/",
     "build:images": "shx rm -rf network-api/networkapi/frontend/_images && shx cp -r source/images network-api/networkapi/frontend/_images",
     "build:js:dev": "node esbuild.config.js -- --node-env development",
     "build:js:prod": "node esbuild.config.js -- --node-env production",
-    "build:sass": "run-s build:sass:clean && run-p build:sass:main build:sass:bg build:sass:donate build:sass:formassembly && run-s optimize:css",
-    "build:sass:clean": "shx mkdir -p network-api/networkapi/frontend/_css",
-    "build:sass:main": "sass source/sass/main.scss network-api/networkapi/frontend/_css/main.compiled.css",
-    "build:sass:bg": "sass source/sass/buyers-guide/bg-main.scss network-api/networkapi/frontend/_css/buyers-guide.compiled.css",
-    "build:sass:donate": "sass source/sass/donate.scss network-api/networkapi/frontend/_css/donate.compiled.css",
-    "build:sass:formassembly": "sass source/sass/formassembly-override.scss network-api/networkapi/frontend/_css/formassembly-override.compiled.css",
-    "build:tailwind": "tailwindcss -i source/postcss/tailwind.css -o network-api/networkapi/frontend/_css/tailwind.compiled.css",
+    "build:css": "run-p build:css:file:**",
+    "build:css:file:main": "sass source/sass/main.scss network-api/networkapi/temp/frontend/_css/main.compiled.css",
+    "build:css:file:bg": "sass source/sass/buyers-guide/bg-main.scss network-api/networkapi/temp/frontend/_css/buyers-guide.compiled.css",
+    "build:css:file:donate": "sass source/sass/donate.scss network-api/networkapi/temp/frontend/_css/donate.compiled.css",
+    "build:css:file:formassembly": "sass source/sass/formassembly-override.scss network-api/networkapi/temp/frontend/_css/formassembly-override.compiled.css",
```

Comment:
Changes

1. Renaming script name from `build:sass:blah` to `build:css:blah` because these scripts are used to compile CSS from *.scss files and not building SCSS.
2. Update file path to the new location.